src/beastie/commands/awesomeness/awesomeness.ts

```diff
@@ -11,7 +11,7 @@ const execute = async (context: CommandContext): Promise<string> => {
 
   let twitchId: string = context.twitchId;
   if (context.para1) {
-    twitchId = await getTwitchId(context.para1);
+    twitchId = (await broadcaster.getProfile())?.id;
```

Comment:
I am *pretty* sure this line introduces a bug where you can't do `!awesomeness <username>` to retrieve the awesomeness for someone other than the broadcaster.

As your change stands, `!awesomeness` still returns your awesomeness, while `!awesomeness SomeoneElse` will always return the broadcaster's awesomeness.

Perhaps you meant the following?
```
    twitchId = (await getTwitchProfile(context.para1))?.id;
```